server/lib/thumbnail.ts

```diff
@@ -61,7 +61,10 @@ function generateVideoMiniature (video: MVideoThumbnail, videoFile: MVideoFile,
   const { filename, basePath, height, width, existingThumbnail, outputPath } = buildMetadataFromVideo(video, type)
   const thumbnailCreator = videoFile.isAudio()
     ? () => processImage(ASSETS_PATH.DEFAULT_AUDIO_BACKGROUND, outputPath, { width, height }, true)
-    : () => generateImageFromVideoFile(input, basePath, filename, { height, width })
+    : () => Promise.all([
+      generateImageFromVideoFile(input, basePath, filename, { height, width }),
+      generateImageFromVideoFile(input, basePath, filename, { height, width }, 10)
```

Comment:
Should be async since the timestamp thumbnails are optional

Reply:
Should I change something there?

Reply:
You could remove the second line, and call the function directly from the controller:

```
...
await generateImageFromVideoFile(...)
generateTimecodeThumbnails(...).catch(err => logger.error)
...
```